### ISSUE TITLE
Missing Pagination Ability

### ISSUE THREAD

**Issue by codebytesfl** [meabed/mongoosastic-ts#38]
In Mongoosastic you can paginate your search by doing the following. 

```js
const results = await bookModel.search(
      {
          query_string: {
              query:  "john",
          },
      },
      {
          size: 10,
          from: 0,
          hydrate: true
      },
  );
```

According to `<IMongoosasticSearchOpts>` though the `size` and `from` are not typed, hydrate is though.

**Comment by codebytesfl** [comment]
The solution is to add `size` and `from` to the typings.

```ts
export interface IMongoosasticSearchOpts {
  suggest?: any;
  sort?: any;
  min_score?: number;
  aggs?: any;
  highlight?: any;
  index?: string;
  routing?: string;
  hydrate?: boolean;
  hydrateWithESResults?: { source?: boolean };
  hydrateOptions?: { lean?: boolean; sort?: string; select?: string };
  from?: number;
  size?: number
}
```

**Comment by meabed** [comment]
Thanks @codebytesfl i will update the typing.

**Comment by meabed** [comment]
Fixed in release  [5.0.5](https://github.com/meabed/mongoosastic-ts/releases/tag/v5.0.5)
